src/services/identity/SitesService.ts

```diff
@@ -89,11 +173,15 @@ class SitesService {
 
     // Commit message created as part of phase 2 identity
     if (message.startsWith("{") && message.endsWith("}")) {
-      const { userId }: { userId: string } = JSON.parse(message)
-      const user = await this.usersService.findById(userId)
+      try {
+        const { userId }: { userId: string } = JSON.parse(message)
+        const user = await this.usersService.findById(userId)
 
-      if (user) {
-        return user.email
+        if (user) {
+          return user.email
```

Comment:
could we only return if `user.email` is a string? this is `string | null | undefined`, and we can do better by returning the commit author's email

Reply:
Good catch! Fixed in fdf07a5.